src/lib/function-utils/cardTranslations.ts

```diff
@@ -16,7 +16,7 @@ const getFrontFace = (card: ScryfallCard): CardFace => {
             type: cardFace.type_line ?? '',
             description: cardFace.oracle_text,
             artist: cardFace.artist,
-            fullImageUri: imageUris.small,
+            fullImageUri: imageUris.normal,
```

Comment:
I'd prefer to keep `imageUris.small` so that I can serve the smaller images and cut down on the bandwidth needed.  Maybe it's worth renaming this to `smallImageUri` and adding in `largeImageUri` which points to `imageUris.normal`?